helpers/handleNetworksValuesApi.ts

```diff
@@ -0,0 +1,26 @@
+const newNetworkObj = (issue) => ({
```

Comment:
redundant, code can live inside the `handleNetwork` function as is as it's not re-used,

```js
const handleNetwork(issues) =>
  issues?
    .filter(i => i?.network)
    .map(({network: {name, logoIcon, colors: {primary})) => 
      ({name, logoIcon, colors: {primary}})
```